src/packages/commands/rat_command.py

```diff
@@ -43,7 +43,31 @@ class NameCollisionException(CommandException):
 _registered_commands = {}  # pylint: disable=invalid-name
 
 # character/s that must prefix a message for it to be parsed as a command.
-PREFIX = config['commands']['prefix']
+PREFIX: str
+
+
+@config_marker
+def rehash_handler(data: Dict):
+    """
+    handle rehash events
+
+    Args:
+        data (Dict): new config object
+    """
+    global PREFIX
+
+    PREFIX = data['commands']['prefix']
+    LOG.debug(f"in rehash, applying new configuration data...")
```

Comment:
[Error] Debug log seems a little unhelpful without any context as to WHICH rehash it's in.

Reply:
context is added by the logger itself. reviewing the log file its quite clear which module emitted this message.